R/MetaAnalysis.R

```diff
@@ -0,0 +1,424 @@
+# Copyright 2021 Observational Health Data Sciences and Informatics
+#
+# This file is part of SkeletonComparativeEffectStudy
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+#' Title
+#'
+#' @param allDbsFolder    Folder on the local file system containing the individual zip files across databases (i.e., sites)
+#' @param maExportFolder  A local folder where the meta-analysis results will be written. If not specified, results will be
+#'                        written to same directory with all other results.
+#' @param maxCores        Maximum number of CPU cores to be used when computing the meta-analyses.
+#' @param method          The meta-analysis method to use.  Options are "bayesian" or "DL" (DerSimonian-Laird).
+#' @param addTraditional  Boolean indicating if traditional meta-analysis (i.e., "DL") results should be added to result (if method is "bayesian").
+#'
+#' @return
+#' Does not return a value, but creates a new zip file in the \code{maExportFolder} for the meta-analyses.
```

Comment:
Add references section with references to:
- DerSimonian R, Laird N. Meta-analysis in clinical trials. Control Clin Trials. 1986 Sep;7(3):177-88. doi: 10.1016/0197-2456(86)90046-2.
- Schuemie M, Chen Y, Madigan D, Suchard M, Combining Cox Regressions Across a Heterogeneous Distributed Research Network Facing Small and Zero Counts. arXiv: 2101.01551, 2021

Reply:
All set here.